### PR TITLE
add support for multiple containers

### PR DESCRIPTION
**Pull request by jessesarn**

### CODE REVIEW

File: plugins/default/lib/openshift.py [hunk 1]
```diff
@@ -44,7 +44,51 @@ def exec_in_pods(project, pods, cmd):
...
+    #         for p in pods if p["status"]["phase"] == "Running"]
+    # ))
+
+
+def get_running_pod_containers(project, selector=None):
+    return _get_running_pod_containers(oc, project, selector)
+
+
 def get_project():
+    # return oc("project", "-q").rstrip("\n")
```
Comment (philbrookes) [inline plugins/default/lib/openshift.py]: Shall we just delete commented code?

Comment (jessesarn) [inline plugins/default/lib/openshift.py]: Yeah, I keep putting that back in so I can easily test the python from my mac without having to put code back into the pod.  I'll remove it before merging.

**Comment by jessesarn** [comment]
@philbrookes Also, I'll fix the flake8 complaints on the test script before merging.

**Comment by philbrookes** [comment]
:+1: Code looks fine, and verified on my local ose32

**Comment by vchepeli** [comment]
LGTM, Motivation and Changes are not presented in PR description!

**Pull request merged**